docs/en/ingest-management/elastic-agent/configuration/inputs/input-winlog.asciidoc

```diff
@@ -0,0 +1,372 @@
+:input-type: winlog
+
+:vista_and_newer: This option is only available on operating systems +
+  supporting the Windows Event Log API (Microsoft Windows Vista and newer).
```

Comment:
Uhm, do we really still support Vista officially? For 7.x we might still support windows 7 for a while (inofficially), but if I'm not mistaken Vista support has been removed from Go maybe 2 or 3 years ago already.